tests/test_integration.py

```diff
@@ -4,11 +4,12 @@
 from sklearn.preprocessing import OrdinalEncoder
 from sklearn.pipeline import Pipeline
 
-from cyclic_boosting import binning, flags, CBPoissonRegressor,\
+from cyclic_boosting import binning, flags,\
     common_smoothers, observers
 from cyclic_boosting.smoothing.onedim import SeasonalSmoother,\
     IsotonicRegressor
 from cyclic_boosting.plots import plot_analysis
+from cyclic_boosting.CBpipeline import CBPoissonRegressorPipeline
```

Comment:
You might add these convenience pipelines to the overall __init__.py, just like the estimators themselves (or maybe instead of those).